src/components/accordion/Accordion.spec.jsx

```diff
@@ -62,24 +64,30 @@ describe('<Accordion>', () => {
   it('expands one item at a time when "allowMultiple" is set to false', () => {
     const accordion = mount(
       <Accordion>
-        <AccordionItem title="Item 1">Accordion Item Description</AccordionItem>
-        <AccordionItem title="Item 2">Accordion Item Description</AccordionItem>
-        <AccordionItem title="Item 3">Accordion Item Description</AccordionItem>
+        <AccordionItem title="Item 1" id="item1">
```

Comment:
@katarzynatobis do we need this change? we don't need `id` if component is uncontrolled

Reply:
It's unnecessary, I added these identifiers for a11y testing and overlooked this change when I moved the a11y test to a separate file.

Reply:
cool, please make sure to remove ids whenever it's unnecessary to not make impression it's needed